src/india_api/cmd/main.py

```diff
@@ -9,6 +10,16 @@
 
 cfg = Config()
 
+sentry_sdk.init(
+    dsn=os.getenv("SENTRY_DSN"),
+    environment=os.getenv("ENVIRONMENT", "local"),
+    traces_sample_rate=1
+)
+
+sentry_sdk.set_tag("app_name", "india_api")
+sentry_sdk.set_tag("version", india_api.__version__)
```

Comment:
I think you should just need to import [this](https://github.com/openclimatefix/india-api/blob/main/src/india_api/internal/service/server.py#L17)  as use this version